tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/metadata.py

```diff
@@ -46,6 +46,27 @@ def lib(ctx: click.Context):
         click.secho(str(e), err=True, fg="red")
         return sys.exit(1)
 
+@test.command(help="Run unit tests for the metadata service orchestrator.")
+@click.pass_context
+def orchestrator(ctx: click.Context):
+    logger.info("Running metadata service orchestrator unit tests...")
+    try:
+        pipeline_success = anyio.run(
+            run_metadata_orchestrator_test_pipeline,
+            ctx.obj["is_local"],
+            ctx.obj["git_branch"],
+            ctx.obj["git_revision"],
+            ctx.obj.get("gha_workflow_run_url"),
+            ctx.obj.get("pipeline_start_timestamp"),
+            ctx.obj.get("ci_context"),
+        )
+        if not pipeline_success:
+            raise dagger.DaggerError("Metadata Service Lib Unit Test Pipeline failed.")
+
+    except dagger.DaggerError as e:
+        click.secho(str(e), err=True, fg="red")
+        return sys.exit(1)
```

Comment:
nit: As we reuse this try  except block quite a lot I think we could use a decorator to decorate the commands with. I could manage the `logger.info` and the `try except`

Reply:
Great idea!